res/qml/Deck/BeatJump.qml

```diff
@@ -0,0 +1,231 @@
+import ".." as Skin
+import Mixxx 1.0 as Mixxx
+import Qt5Compat.GraphicalEffects
+import QtQuick 2.12
+import QtQuick.Shapes
+import QtQuick.Controls 2.12
+import "../Theme"
+
+Rectangle {
+    id: root
+
+    required property string group
+
+    color: '#626262'
+
+    Label {
+        anchors.top: parent.top
+        anchors.horizontalCenter: parent.horizontalCenter
+        text: "Beatjump"
+        color: '#3F3F3F'
+    }
+
+    Skin.ControlButton {
+        id: jumpBackButton
+        anchors {
+            top: parent.top
+            topMargin: 22
+            left: parent.left
+            leftMargin: 6
+        }
+
+        implicitWidth: 50
+        implicitHeight: 26
+
+        group: root.group
+        key: "beatjump_backward"
+        contentItem: Item {
+            anchors.fill: parent
+            Shape {
+                anchors.centerIn: parent
+                width: 21
+                height: 14
+                antialiasing: true
+                layer.enabled: true
+                layer.samples: 4
+                ShapePath {
+                    fillColor: '#D9D9D9'
+                    startX: 0; startY: 1
+                    PathLine { x: 1; y: 1 }
+                    PathLine { x: 1; y: 7 }
+                    PathLine { x: 10; y: 1 }
+                    PathLine { x: 10; y: 7 }
+                    PathLine { x: 20; y: 1 }
+                    PathLine { x: 21; y: 1 }
+                    PathLine { x: 21; y: 13 }
+                    PathLine { x: 20; y: 13 }
+                    PathLine { x: 10; y: 7 }
+                    PathLine { x: 10; y: 13 }
+                    PathLine { x: 1; y: 7 }
+                    PathLine { x: 1; y: 13 }
+                    PathLine { x: 0; y: 13 }
+                    PathLine { x: 0; y: 1 }
+                }
```

Comment:
Shouldn't these be SVG files?

Reply:
I fill like using `ShapePath` for simple form is a little easier as it allow us to do complex animation (e.g a play button morphing into a pause button). Tho SVG support animation, I don't think we could animate them easily in QML.

Shall I create an issue so we can discuss this further and commit to either using this form for simple shapes or SVG across the board before launch? It should be pretty easy to convert to SVG later if we decide to.

Reply:
#15388